src/Tribe/Admin/Move_Tickets.php

```diff
@@ -672,8 +672,8 @@ public function move_tickets( array $ticket_ids, $tgt_ticket_type_id, $src_event
 			if ( 'Tribe__Tickets__RSVP' === $ticket['provider'] ) {
 				$src_stock ++;
 				$tgt_stock --;
-				update_post_meta( $src_ticket_type_id, '_stock', $src_stock );
-				update_post_meta( $tgt_ticket_type_id, '_stock', $tgt_stock );
+				update_post_meta( $src_event_id, '_stock', $src_stock );
+				update_post_meta( $tgt_event_id, '_stock', $tgt_stock );
```

Comment:
@pattihis i think that the meta `_stock` is relevant only at a ticket. Not the event itself. The event has different meta to evaluate its total capacity (see global stock consts, and tickets handler key_capacity property.